src/pkg/packaging/windows/sharedframework/bundle.wxs

```diff
@@ -27,7 +27,7 @@
 
     <swid:Tag Regid="microsoft.com" InstallPath="[DOTNETHOME]" />
 
-    <Variable Name="DOTNETHOME" Type="string" Value="[ProgramFiles6432Folder]dotnet" bal:Overridable="no" />
+    <Variable Name="DOTNETHOME" Type="string" Value="[$(var.Program_Files)]dotnet" bal:Overridable="no" />
```

Comment:
Why are we changing this to install in "Program Files" instead of the current "Program Files (x86)"?